modules/sw360/sw360-client/src/main/java/org/eclipse/sw360/antenna/sw360/client/rest/SW360Client.java

```diff
@@ -166,7 +193,7 @@ protected String resourceUrl(String... paths) {
 
         CompletableFuture<T> futRequest = getTokenProvider().doWithToken(accessToken ->
                 getClientConfig().getHttpClient().execute(accessToken.tokenProducer(producer),
-                HttpUtils.checkResponse(processor, tag)));
+                        HttpUtils.checkResponse(processor, tag)));
```

Comment:
Small thing: This seems to be a reformatting issue

Reply:
Fixed.